applications/brain_dataset_tutorial.md

```diff
@@ -0,0 +1,77 @@
+
+A very important concept to bear in mind when going through this tutorial is the [Lazy Loading](https://en.wikipedia.org/wiki/Lazy_evaluation), i.e., only loading images to memory when the position slider on napari maps to that particular image, which is very of use, when using large datasets.
+
+#### So how can we lazily load an image using dask?
+First, we import delayed from dask, which is the main function responsible for handling __Lazy Loading__. Here we defined a function *load_image()* that will lazy load the image when called given, the image name as a parameter. 
+```
+
+from dask import delayed
+import dask.array as da
+def load_image(image_name):
+	delayed_image_load = delayed(io.imread)
+	return da.from_delayed(delayed_image_load(image_name), dtype='float64', shape=(1760, 1760))
+```
+#### Getting the data
+
+You can download the dataset from [here](https://datadryad.org/stash/downloads/file_stream/223913). A ZIP file will be downloaded, after unzipping the compressed dataset, the `.tiff` images will be extracted to the `SYNTHESIZED_TIFF_Images_Raw/Synthesized_FLASH25_100um_TIFF_Axial_Images/` directory. Now your dataset is ready to be used!
+
+
+
+#### Reading images from a folder
+To load images from a folder we use io.imread, but first we need to  import io from skimage.
+```
+ delayed_image_load = delayed(io.imread)
+```
+
+And to read images from a folder, you should import *glob* to list all the images in the dataset folder, and this is how it is used:
+```
+images_names = glob.glob(
+'SYNTHESIZED_TIFF_Images_Raw/Synthesized_FLASH25_100um_TIFF_Axial_Images/Synthesized_FLASH25_Axial_*.tiff')
+images_names = sorted(images_names)
+```
+Here we use the `sorted` function, to sort the images as, glob returns the images unsorted.
+
+
+#### Using dask.stack() to concatenate images
+
+Dask.array.stack (seq, axis=0) used to stack many existing dask arrays into a new array, creating a new dimension (axis=0 by default), And this is how it is used:
+```
+images = da.stack(load_image(image_name) for image_name in images_names) 
+```
+
+
+
+Here's the complete code:
+``` 
+from dask import delayed
+import dask.array as da
+import napari
+import glob
+from skimage import io
+
+def load_image(image_name):
+	delayed_image_load = delayed(io.imread)
+	return da.from_delayed(delayed_image_load(image_name), dtype='float64', shape=(1760, 1760))
+
+if __name__ == '__main__':
+    # Load images
+    images_names = glob.glob(
+        'SYNTHESIZED_TIFF_Images_Raw/Synthesized_FLASH25_100um_TIFF_Axial_Images/Synthesized_FLASH25_Axial_*.tiff')
+    images_names = sorted(images_names)
+    images = da.stack(load_image(image_name) for image_name in images_names).reshape(len(images_names), 1760, 1760)
+    with napari.gui_qt():
+        napari.view_image(images) 
+```
+
+__And here's the final output!__
+![Alt text](https://media.giphy.com/media/LO3gPbCs5AxApjAehW/giphy.gif)
```

Comment:
suggest add additional so that the image is on it's own line

Reply:
It might also be kind of cool to have a gif of what the dataset looks like when you click the "3D mode" button at the bottom right of the napari viewer.

Reply:
@GenevieveBuckley isn't this dataset 32GB? I think the viewer will explode if you click that button. =P

Reply:
Whoops, I hadn't finished downloading the data so I didn't think about that. Apparently unzipped it's 3.7GB (not 32.0GB), and yes, it freezes the viewer completely. So that's a terrible idea, I take it back

Reply:
Oh, but now I've found the [data webpage](https://doi.org/10.5061/dryad.119f80q) I see where you got 32GB from - that must be the size of all the datasets from that experiment together, not just the single one visualized in this example.

Reply:
I wonder why the original dataset page lists it as 32GB... But yeah, perhaps making this render in 3D (via downsampling) is best left as an exercise for another time. =)

Reply:
Well there's way more stuff in the dataset page than just the one folder of tiffs we look at @jni. 

They have multiple .nii.gz files for axial, sagittal, and coronal planes. Each of those is also uploaded as a folder of tiffs (duplicating the data), and also uploaded as a downsampled version with 200 micron resolution (duplicating data again). They've also uploaded and saved the scan protocol information, as well as all the mp4 recordings, and some more stuff I haven't mentioned. All in all, that adds up to 32GB.

It makes it super convenient for us to just download the format we prefer, instead of having to waste effort converting from .nii files. So I'm pleased the authors went to all that effort.